build/bazel/remote/execution/v2/remote_execution.proto

```diff
@@ -207,6 +207,21 @@ service ActionCache {
 // `{instance}/uploads/{uuid}/blobs/{hash}/{size}/foo/bar/baz.cc`. Anything
 // after the `size` is ignored.
 //
+// For servers that support compression, clients can upload compressed data
```

Comment:
I guess the "For servers that support compression," is redundant and can be removed (similarly below).